recipe/meta.yaml

```diff
@@ -10,57 +10,75 @@ source:
   sha256: 36763d234243ff5c7b5646a9f77706623e7ba5b08770f9020ed759c61a5b0693
 
 build:
+  number: 2
   skip: true  # [py>38]
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
 
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - llvm-openmp  # [osx]
-  host:
-    - python
-    - cython >=0.29
-    - numba >=0.50,<0.55
-    - numpy >=1.19
-    - pandas >=1.1.0,<1.4
-    - pip
-    - setuptools
-    - scikit-learn >=0.24.0
-    - statsmodels >=0.12.1
-    - wheel
-  run:
-    - python
-    - {{ pin_compatible('numpy') }}
-    - pandas >=1.1.0,<1.4
-    - scikit-learn >=0.24.0
-    - statsmodels >=0.12.1
-    - numba >=0.50,<0.55
-  run_constrained:
```

Comment:
The `sktime` package will still need the `run_constrained` for all optional deps which have a version constraint - i.e. this shouldn't be removed and the constraints should match those in `sktime-all-extras`

Reply:
Ah I see what you mean. I wasn't too sure what `run_constrained` meant at first. I will add back and have it match the contraines in `all-extras`. Also, I'm adding `!=1.8.0` for pmdarima because that's what the `setup.py` says.